dm/dm/master/dm-master.toml

```diff
@@ -39,5 +39,9 @@ rpc-timeout = "30s"
 # refilled at rate `rpc-rate-limit` tokens per second. Note `rpc-rate-limit`
 # is float64 type, so remember to add a decimal point and one trailing 0 if its
 # literal value happens to be an integer.
-rpc-rate-limit = 10.0
 rpc-rate-burst = 40
+rpc-rate-limit = 10.0
+
+# some experimental features
+[experimental-features]
```

Comment:
Just `experimental` to keep the same with tidb?
https://github.com/pingcap/tidb/blob/master/config/config.toml.example#L485

Reply:
added in https://github.com/pingcap/ticdc/pull/3266/commits/cc7ef4d4e0e1c6d18049b71d5b7a1dc2064d0414